SonarQube.TeamBuild.PreProcessor/RulesetGenerator.cs

```diff
@@ -7,20 +7,27 @@
 
 using System;
 using System.Collections.Generic;
-using System.Diagnostics.Contracts;
 using System.IO;
 using System.Linq;
-using System.Net;
 
 namespace SonarQube.TeamBuild.PreProcessor
 {
-    public class RulesetGenerator : IRulesetGenerator
+    public static class RulesetGenerator
```

Comment:
No longer needs to be an interface as the abstraction is provided by the ISonarQubeServer interface instead.